.ci/install_kata_image.sh

```diff
@@ -5,10 +5,23 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-set -e
+set -o errexit
+set -o nounset
+set -o pipefail
 
 cidir=$(dirname "$0")
 
+tmp_dir=$(mktemp -d -t kata-image-install.XXXXXXXXXX)
+readonly ROOTFS_DIR="${tmp_dir}/rootfs"
+export ROOTFS_DIR
+
+finish() {
+  [ -d "${ROOTFS_DIR}" ] && [[ "${ROOTFS_DIR}" = *"rootfs"* ]] && sudo rm -rf "${ROOTFS_DIR}"
```

Comment:
@jodh-intel @grahamwhaley  does it looks better?

Reply:
What we'd call 'belt and braces' :-)
Wacky use of *'s and those [['s
presuming you tested it (which I strongly suspect you did), then
lgtm

Reply:
@grahamwhaley one day you will get used to them 👍